### DISCUSSION TITLE
laser going over limits and wont go the right speed

### DISCUSSION THREAD

**Discussion by Lunatic11x** [arkypita/LaserGRBL#1528] [Q&A]
When i put the size of the image in and the speed and power it mover supper slow and goes over limits. it just started doing this a couple hours ago jun the middle of a project. example if i tell it to move 10mm it moves 30mm.

**Comment by gmmanonymus111** [comment]
I'd check `$100` & `$101` stepper resolution settings, it might be set incorrectly. This should fix your ratio issue.
For slow speed, check `$110` & `$111` maximum feed rate and/or `$120` `$121` acceleration settings.

**Reply by Lunatic11x** [reply]
Would you happen to know how do I figure out what its supposed to be set to? I went over the configuration and it all set how they recommended.
